source/patterns/@aws-solutions-constructs/core/lib/elasticsearch-defaults.ts

```diff
@@ -14,15 +14,13 @@
 import * as elasticsearch from '@aws-cdk/aws-elasticsearch';
 import * as cognito from '@aws-cdk/aws-cognito';
 import * as iam from '@aws-cdk/aws-iam';
-import * as lambda from '@aws-cdk/aws-lambda';
 import * as ec2 from '@aws-cdk/aws-ec2';
 import * as cdk from '@aws-cdk/core';
 
 export interface BuildElasticSearchProps {
   readonly identitypool: cognito.CfnIdentityPool,
   readonly userpool: cognito.UserPool,
   readonly cognitoAuthorizedRoleARN: string,
-  readonly lambdaFunction?: lambda.Function,
   readonly serviceRoleARN?: string,
```

Comment:
Doesn't removing all the lambda stuff here feel right now that you're doing it? :-)